zest/releaser/bzr.py

```diff
@@ -24,10 +24,10 @@ def name(self):
         return dir_name
 
     def available_tags(self):
-        tag_info = execute_command('bzr tags')
-        tags = [line[:line.find(' ')] for line in tag_info.split('\n')]
+        tag_info = execute_command([u'bzr', u'tags'])
+        tags = [line[:line.find(u' ')] for line in tag_info.split(u'\n')]
         tags = [tag for tag in tags if tag]
-        logger.debug("Available tags: %r", tags)
+        logger.debug(u"Available tags: {0!r}".format(tags))
```

Comment:
[PEP 3101](https://www.python.org/dev/peps/pep-3101/) & [str.format in the Python 3 docs](https://docs.python.org/3/library/stdtypes.html#str.format).

The [object.\__format\__ special method](https://docs.python.org/3/reference/datamodel.html#object.__format__) is also particularly fun.